epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -127,9 +130,19 @@ private void processAttribute(Element attribute,
     TypeName type = TypeName.get(attribute.asType());
     boolean hasSuper = hasSuperMethod(classElement, name);
     boolean hasFinalModifier = attribute.getModifiers().contains(FINAL);
+    boolean packagePrivate = isFieldPackagePrivate(attribute);
     helperClass.addAttribute(
         new AttributeInfo(name, type, attribute.getAnnotationMirrors(),
-            attribute.getAnnotation(EpoxyAttribute.class), hasSuper, hasFinalModifier));
+            attribute.getAnnotation(EpoxyAttribute.class), hasSuper, hasFinalModifier,
+            packagePrivate));
+  }
+
+  /**
+   * Checks if the given field has package-private visibility
+   */
+  private boolean isFieldPackagePrivate(Element attribute) {
+    Set<Modifier> modifiers = attribute.getModifiers();
+    return !modifiers.contains(PUBLIC) && !modifiers.contains(PROTECTED);
```

Comment:
Check for `!Private` too? I know we enforce a field can't be private elsewhere, but in the context of just this method it seems like we should also check it for this method to be correct